server/database/queries/get_session_responses_number.js

```diff
@@ -1,7 +1,7 @@
 const Response = require("../models/Response");
 
-const getSessionResponsesNumber = (sessionId = "5bf3101ca8d8483454ce5da1") => new Promise((resolve, reject) => {
-  Response.count({ session: sessionId })
+const getSessionResponsesNumber = (sessionId, surveyType) => new Promise((resolve, reject) => {
+  Response.count({ session: sessionId, surveyType })
     .then(reponsesNumber => resolve({ reponsesNumber }))
     .catch(reject);
```

Comment:
Do you need to reject if session is not found? It might be better to resolve with an empty object to show that no response was found, without throwing an error